### ISSUE TITLE
How do we deal with more than one accused?

### ISSUE THREAD

**Issue by robertocarroll** [robertocarroll/icc-alpha#29]
Often more than one accused per case. 

![case-page-top 001](https://f.cloud.github.com/assets/1565953/1464745/d9387d9a-4551-11e3-8e4d-94244e752b68.jpg)

![case-page-top 002](https://f.cloud.github.com/assets/1565953/1464746/dc099040-4551-11e3-88b1-f819b2cf8933.jpg)

![case-page-top 003](https://f.cloud.github.com/assets/1565953/1464748/dfe89256-4551-11e3-8052-5f71ebbb8a82.jpg)

**Comment by robertocarroll** [comment]
Does each accused need his or her own page?

No. Link to Warrant of arrest will go in description.

**Comment by robertocarroll** [comment]
What about being able to browse by accused? 

Suggest: 

Accused A-Z list of accused
http://icc-cpi.int/accused

See https://github.com/robertocarroll/icc-alpha/issues/3 for details

**Comment by robertocarroll** [comment]
I am starting to think that each accused needs a page. 

Let's take more than one accused as the standard case. If each accused has a page with name, description, case, charges, we can use that in the case page and we can use it in an A-Z list of the accused. 

So in the sketch above, we would eliminate the first one and always have: 

Case Name
Case description 

Accused name (Link to accused page)
Accused description 

This deals with one or many accused per case and it creates a clear distinction between the case and the accused. 

The page would sit below the case page: 

http://icc-cpi.int/situtation/case/accused

http://icc-cpi.int/kenya/ruto-sang/ruto 

Any thoughts?

**Comment by robertocarroll** [comment]
I've done a mock up for two accused using Ruto and Sang. What information needs to be available in the case description? c.f. specific information about each accused? 

![case-page-top-two-accused-d4-r2](https://f.cloud.github.com/assets/1565953/1522338/6fc5ff08-4ba6-11e3-9181-d78b498f84ed.jpg)

**Comment by taslaman** [comment]
This could be a good way to present the case. What would we put in the accused page? We don't usually have that much about individuals.

In any case, Michelle and Daphny shoild

**Comment by taslaman** [comment]
Sorry I'm typing on my phone. So, Michelle and Daphnie should be consulted about this. If we don't make separate pages, we can still be able to create accused pages. We can put informatiob about the accused in a separate list and then have some of those entries displayed on the accused page using a content rollup web part. The list needs to include the link to the case page in order to link the accused name to the case. Sorting of the entries might be a problem but we need to see if this would be a good way to go.

**Comment by robertocarroll** [comment]
I think this information from the case information sheet would work on an accused page:

![rutokosgeysangeng pdf page 1 of 3](https://f.cloud.github.com/assets/1565953/1566479/a25fb13c-508f-11e3-8fbd-de2ffb9d842d.jpg)
![preview](https://f.cloud.github.com/assets/1565953/1566494/d0ef882e-508f-11e3-8003-e33caf8f434b.jpg)

**Comment by robertocarroll** [comment]
I think we've come an agreement on this approach. Please re-open the issue if not. Thanks.

**Comment by robertocarroll** [comment]
Re-opening this issue because it is not resolved.

**Comment by robertocarroll** [comment]
Kyle experimented with an accordion pattern but it didn't work out. Here's his note:

"I did put that in but honestly it seemed to take up so much room that I ended up taking it back out.  I think for now simply stacking the accused images/text on top of each other might be the best option. If we want to experiment with accordions and/or other treatments we may have to consider moving this content elsewhere on the page to make more room for it."
